aws/resource_aws_secretsmanager_secret.go

```diff
@@ -154,6 +175,24 @@ func resourceAwsSecretsManagerSecretRead(d *schema.ResourceData, meta interface{
 	d.Set("description", output.Description)
 	d.Set("kms_key_id", output.KmsKeyId)
 	d.Set("name", output.Name)
+
+	pIn := &secretsmanager.GetResourcePolicyInput{
+		SecretId: aws.String(d.Id()),
+	}
+	log.Printf("[DEBUG] Reading Secrets Manager Secret policy: %s", pIn)
+	pOut, err := conn.GetResourcePolicy(pIn)
+	if err != nil {
+		return fmt.Errorf("error reading Secrets Manager Secret policy: %s", err)
+	}
+
+	if pOut.ResourcePolicy != nil {
+		policy, err := structure.NormalizeJsonString(*pOut.ResourcePolicy)
+		if err != nil {
+			return errwrap.Wrapf("policy contains an invalid JSON: {{err}}", err)
```

Comment:
Nitpick: This is likely copy paste from other code, but we prefer to use the standard library `fmt.Errorf()` for returning error messages instead of `errwrap.Wrapf()` which has functionality we do not need in this case. 👍  At some point we'll get around to cleaning up all the other references to `errwrap.Wrapf()`.